package.json

```diff
@@ -2,15 +2,19 @@
   "name": "outline-client",
   "productName": "Outline",
   "scripts": {
-    "action": "node ./src/build/run_action.mjs",
     "action:help": "npm run action list",
     "action:list": "npm run action list",
+    "action": "node ./src/build/run_action.mjs",
     "clean": "rm -rf build node_modules www platforms plugins",
-    "format": "pretty-quick --staged --pattern \"**/*.{cjs,mjs,html,js,json,md,ts}\"",
     "format:all": "prettier --write \"**/*.{cjs,mjs,html,js,json,md,ts}\"",
-    "lint": "npm run lint:ts",
+    "format": "pretty-quick --staged --pattern \"**/*.{cjs,mjs,html,js,json,md,ts}\"",
     "lint:ts": "eslint 'src/**/*.ts'",
-    "reset": "npm run clean && npm ci"
+    "lint": "npm run lint:ts",
+    "reset": "npm run clean && npm ci",
+    "start:electron": "OUTLINE_DEBUG=true electron .",
```

Comment:
Shall we start from current folder `.` or the build output folder ([manager](https://github.com/Jigsaw-Code/outline-server/blob/master/src/server_manager/electron_app/start.action.sh#L19) starts from the build output folder)?

Reply:
Ah, the paths are wrong currently, but we weren't sure if they would they work on windows as written

Reply:
You need to pass the electron build output directory to the electron call.